src/main/java/de/tum/in/www1/artemis/repository/UserRepository.java

```diff
@@ -318,4 +357,10 @@ default Long countUserInGroup(String groupName) {
     default void updateUserNotificationReadDate(long userId) {
         updateUserNotificationReadDate(userId, ZonedDateTime.now());
     }
+
+    @Query("select user from User user left join fetch user.organizations")
+    List<User> findAllWithEagerOrganizations();
```

Comment:
This is not a good query. It might load 15.000 users, we should avoid this

Reply:
Removed and replaced by **findAllMatchingEmailPattern**